Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -136,6 +136,68 @@ protected override void ProcessRecord()
 
     /// <summary>
     /// <para type="synopsis">
+    /// Updates information in an existing dataset.
+    /// </para>
+    /// <para type="description">
+    /// Updates information in an existing dataset. If no Project is specified, the default project will be used. 
```

Comment:
I put it in caps because it was referring to the flag -Project.  should I put the dash in there to make it more clear?